packages/video_player/video_player/ios/Classes/FLTVideoPlayerPlugin.m

```diff
@@ -375,7 +377,7 @@ - (int64_t)position {
 }
 
 - (int64_t)duration {
-  return FLTCMTimeToMillis([[_player currentItem] duration]);
+  return FLTCMTimeToMillis([[[_player currentItem] asset] duration]);
```

Comment:
https://openradar.appspot.com/radar?id=4968600712511488

Reply:
You may want to put this link in a comment over this line.

Reply:
Done.